en/docs/develop/restapis/association.yaml

```diff
@@ -67,17 +72,17 @@ paths:
   /me/associations:
     post:
       description: |
-         This API is used to associate a user to the authenticated user. For example, if it is required to associate  my user account to the user account of 'john', this endpoint can be used. userId and the password are required to associate the accounts.
+        This API is used to associate a user to the authenticated user. For example if it is required associate  my user account to the user account of 'john', this endpoint can be used. userId and the password are required to associate the accounts.
 
-          <b>Permission required:</b>
-          * /permission/admin/login
+         <b>Permission required:</b>
+         * /permission/admin/login
       summary: |
         Associate a user to the authenticated user
       parameters:
         -
           name: association
           in: body
-          description:  User details to be associated. userId should be the fully qualified username of the user.
+          description:  User details to be associated. userId should be the fully qalified username of the user.
```

Comment:
qualified

Reply:
Addressed in 861bcaec1512c2d43091b40273eaf2c2dce11f3e